appengine/pond/duck/js/duck.js

```diff
@@ -193,6 +191,21 @@ Pond.Duck.init = function () {
   Pond.Duck.ignoreEditorChanges_ = false;
 };
 
+/**
+ * Add the correct classes to make the tab at the sepcified index look selected.
+ * @param {Array.<!Element>} tabs The list of tabs.
```

Comment:
Instead use the global variable:
`Pond.Duck.editorTabs`

Reply:
done.